layouts/partials/head/twitter.html

```diff
@@ -0,0 +1,15 @@
+{{ if .Site.Params.Settings.pageTwitter}}
+<meta name="twitter:site" content="{{ .Site.Params.Settings.pageTwitter}}">
+{{ else }}
+<meta name="twitter:site" content="{{ .Site.Params.social.twitter}}">
+{{ end }}
+<meta name="twitter:creator" content="{{ .Site.Params.social.twitter}}">
```

Comment:
Move `twitter:creator` tag after the [`/layouts/partials/head/opengraph.html` file's 64th line](https://github.com/MunifTanjim/minimo/blob/cd04fa47c0278dc88890c3b583f20a0afd88a56b/layouts/partials/head/opengraph.html#L64).

Let's use the [author's profile data](https://minimo.netlify.com/docs/authors/) for it.

```golang
    {{- with $author.social.twitter -}}
      <meta name='twitter:creator' content='@{{ . }}'>
    {{- end -}}
```